### ISSUE TITLE
Pectoralis major tear

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11013]
**Preferred term label:**
Pectoralis major tear

**Synonyms**

**Definition (free text, please give PubMed ID)**
Ultrasound or MRI of the chest/upper arm showing partial or complete tear of the pectoralis major tendon, muscle, or myotendinous junction

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Pectoralis major rupture

**Your nano-attribution (ORCID)**
https://radiopaedia.org/articles/pectoralis-major-injury

**Comment by pnrobinson** [comment]
@MickeySegal 
added as 
HP:0020335
Pectoralis major tear
